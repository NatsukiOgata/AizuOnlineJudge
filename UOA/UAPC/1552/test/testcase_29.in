1000 5 12 4 0 14 8 6 10 7 16 19 13 0 13 0 7 1 17 0 13 9 10 5 7 8 9 9 16 12 17 10 10 12 14 14 11 7 5 14 1 19 19 17 17 19 12 16 3 9 0 7 8 3 14 3 3 14 18 18 17 3 12 10 1 4 11 17 15 16 9 19 12 5 6 7 2 12 10 16 1 3 6 4 5 7 7 10 14 10 17 1 13 13 18 11 14 9 1 1 9 12 7 3 17 19 6 5 2 19 11 10 6 16 3 13 13 18 16 8 14 10 2 10 14 18 11 5 10 13 5 2 3 6 4 5 0 8 13 0 3 2 8 17 5 8 7 3 1 4 0 7 15 5 19 10 15 11 11 15 15 4 3 3 4 1 4 7 12 0 12 14 17 13 11 5 7 12 16 6 13 7 12 0 2 13 17 6 14 15 1 13 8 4 4 16 13 8 9 3 19 10 1 8 17 6 14 1 15 12 9 3 9 15 1 5 5 5 13 18 11 13 9 9 14 13 7 9 1 6 15 19 9 8 2 13 17 1 12 4 2 13 5 1 14 17 11 15 13 10 9 11 5 0 12 18 7 8 14 0 4 15 13 0 7 11 2 18 3 12 1 5 2 15 19 17 4 12 18 13 1 8 6 12 1 13 0 1 4 17 15 5 0 5 18 11 1 14 12 16 19 16 13 17 12 4 10 5 4 3 1 5 17 10 10 2 19 2 19 3 0 18 0 5 13 11 14 6 15 1 0 4 16 2 14 18 19 1 5 1 15 8 5 18 11 19 13 2 2 2 7 15 18 1 8 18 15 5 17 2 6 4 8 10 1 3 12 16 1 19 10 15 1 7 9 11 19 10 18 19 17 7 17 6 14 13 12 18 15 16 8 17 14 6 8 4 17 7 7 7 11 5 16 13 15 1 7 4 3 6 17 7 1 16 15 10 0 3 7 8 6 15 19 8 3 9 2 9 8 19 7 2 18 14 13 6 4 14 5 12 1 8 12 11 12 4 9 1 1 4 12 15 11 7 0 13 17 0 12 6 8 14 8 10 13 13 9 10 12 0 0 9 14 19 10 13 18 1 2 9 11 6 7 17 10 11 18 11 9 3 3 10 0 10 17 7 16 15 16 6 12 13 1 3 13 14 0 9 9 8 15 19 0 18 14 7 4 11 10 18 19 15 17 13 15 10 14 8 18 18 19 5 4 12 7 7 15 13 17 16 9 2 8 15 7 16 7 16 15 13 0 18 7 1 0 17 8 11 19 18 12 14 12 8 2 4 5 4 18 4 4 13 19 12 5 16 3 12 9 15 9 12 0 16 14 3 18 19 12 16 19 6 3 6 19 5 9 17 3 9 5 8 19 8 1 17 17 6 0 9 12 8 1 1 14 10 12 5 8 6 13 9 2 13 19 8 0 10 12 7 17 19 9 10 8 13 1 6 15 3 10 4 2 8 10 5 6 9 14 2 11 11 19 12 8 9 12 10 13 5 4 19 18 14 19 19 5 6 18 14 2 15 3 4 12 4 7 3 13 12 18 1 19 6 0 6 18 5 9 5 13 7 18 2 12 19 7 10 15 2 9 17 17 10 15 10 13 16 1 1 0 17 4 10 7 7 6 17 2 14 10 11 9 15 10 8 8 8 11 4 19 17 6 10 12 9 3 12 13 11 10 9 19 1 17 0 15 9 10 1 19 3 16 19 17 12 3 11 13 18 13 3 0 16 7 6 6 18 16 15 9 19 11 17 19 12 11 19 0 10 2 16 18 11 5 18 2 15 0 15 15 15 13 3 16 4 16 12 16 10 0 18 11 8 9 11 15 6 8 3 9 15 0 16 3 3 4 14 13 16 3 18 3 3 8 6 15 13 1 2 10 2 11 14 12 0 18 5 9 10 19 6 13 16 11 4 5 2 14 9 6 18 3 2 12 7 7 11 12 0 1 0 0 4 18 1 3 11 7 13 4 13 1 7 8 12 0 0 2 12 8 13 8 0 4 18 16 15 6 14 5 0 7 13 4 19 6 12 2 1 16 11 2 7 12 16 8 9 17 11 15 10 13 16 16 3 14 14 8 6 4 18 13 18 12 6 15 17 16 10 9 16 14 1 15 16 10 4 9 9 16 9 17 8 3 8 15 12 18 14 13 18 14 10 17 18 18 4 9 12 0 8 12 16 0 6 6 3 6 19 9 1 15 19 18 12 8 16 11 3 13 9 13 4 5 6 5 14 7 13 0 0 9 18 0 1 19 6 1 1 1
