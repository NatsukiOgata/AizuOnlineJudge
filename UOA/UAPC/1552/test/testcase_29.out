14 24 11 308 297
